.travis.yml

```diff
@@ -41,6 +41,7 @@ env:
   # SystemVerilog tools
   - PACKAGE=zachjs-sv2v         CONDA_CHANNELS="conda-forge"
   - PACKAGE=odin_II             CONDA_CHANNELS="pkgw-forge,conda-forge"
+  - PACKAGE=tree-sitter-verilog CONDA_CHANNELS="pkgw-forge,conda-forge"
```

Comment:
Does this *actually* need `pkgw-forge` and `conda-forge`?

Reply:
I forgot to remove them, it builds just fine with the default channel only